components/HTTP/src/Websocket.cpp

```diff
@@ -302,4 +302,36 @@ void Websocket::checkConnection()
     if (m_client.size() != 0) {
         Websocket::pong(req_handler);
     }
-}
\ No newline at end of file
+}
+
+void Websocket::sendUart(httpd_ws_frame_t ws_pkt)
+{
+    uint8_t buf[ws_pkt.len];
```

Comment:
```suggestion
    uint8_t* buf = malloc(ws_pkt.len);
```

Reply:
Don't use a Variable-Lengt-Array as this can [overflow the stack](https://stackoverflow.com/questions/214741/what-is-a-stackoverflowerror) when receiving a packet too big

Reply:
Using malloc fixes this, and works the same :wink: